src/todolist.js

```diff
@@ -1 +1,82 @@
+class TodoList {
+  constructor() {
+    this.todos = []
+  }
 
+  createUniqueId() {
+    let uniqueId = this.todos.length + 1
+
+    while (this.todos.filter((todo) => todo.id === uniqueId)[0]) {
+      uniqueId += 1
+    }
+
+    return uniqueId
+  }
+
+  newTodo(text) {
+    if (!text || text === '') return false
+
+    const newId = this.createUniqueId()
+    const newTodo = {
+      id: newId,
+      text: text,
+      checked: false
+    }
```

Comment:
Minor, but remember to pay close attention to the details of the requirements you're given. A `checked` boolean functionally does the same thing, but the requirement explicitly asks for a status property with a string value